app/src/main/java/com/sdp/movemeet/models/User.java

```diff
@@ -17,40 +16,45 @@
     private String idUser;
     private String imageId;
     private String documentPath;
+    private ArrayList<String> createActivitiesId;
```

Comment:
Could you just please rename the name of this ArrayList from createActivitiesId to createdActivitiesId? I think that the missing "d" is a typo here...

Reply:
Yes I did it in my last commit !

Reply:
Nice thank you!